_po/ja/how-to/laravel.po

```diff
@@ -687,6 +1032,24 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"/**\n"
+" * @extends \\Illuminate\\Database\\Eloquent\\Factories\\Factory<\\App\\Models\\Post>\n"
+" */\n"
+"class PostFactory extends Factory\n"
+"{\n"
+"    /**\n"
+"     * Define the model's default state.\n"
+"     *\n"
+"     * @return array<string, mixed>\n"
+"     */\n"
+"    public function definition(): array\n"
+"    {\n"
+"        return [\n"
+"            //\n"
+"        ];\n"
+"    }\n"
+"}\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.